recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
         - def_snprintf.patch  # [win]
 
 build:
-    number: 3
+    number: 1
```

Comment:
> There may be a solver bug involved.

I'm personally not convinced and suspect it was a fluke, but no harm in trying. 😄